TezosKit/Client/TezosClient.swift

```diff
@@ -366,20 +372,18 @@ public class TezosClient {
     // check if any of the operations to perform requires the address to be revealed. If so,
     // prepend a reveal operation to the operations to perform.
     if operationMetadata.key == nil {
-      for operation in operations {
-        if operation.requiresReveal {
-          let revealOperation = RevealOperation(from: source, publicKey: keys.publicKey)
-          mutableOperations.insert(revealOperation, at: 0)
-          break
-        }
+      for operation in operations where operation.requiresReveal {
+        let revealOperation = RevealOperation(from: source, publicKey: keys.publicKey)
+        mutableOperations.insert(revealOperation, at: 0)
+        break
```

Comment:
if let first = operations.first(where: { $0.requiresReveal }) {
  ....
}

would be shorter to write than a for loop where you immediately break

Reply:
Done